From private repo

v-alje opened the pull request.

azurecla commented:
Hi __@jeffstokes72__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <p>
        It looks like you're working at Microsoft (jeffstok). If you're full-time, we DON'T require a contribution license agreement.
    </p>
    <p>
        If you are a vendor, or work for Microsoft Open Technologies, DO please sign the electronic contribution license agreement. It will take 2 minutes and there's no faxing! https://cla.azure.com.
    </p>

TTYL, AZPRBOT;

jeffstokes72 commented:
Wrong repo, sorry closing and doing this right.

tdykstra committed:
add precompiled c#

tdykstra committed:
fix links

itechedit committed:
edit pass: luis-resources-faq

tdykstra committed:
c# --> .net

tdykstra committed:
jehollan feedback

tdykstra committed:
Merge branch 'master' into cscstg

tdykstra committed:
minor corrections

itechedit committed:
Merge branch 'master' of https://github.com/microsoft/azure-docs-pr into luis-resources-faq

itechedit committed:
edit pass: luis-resources-faq

itechedit committed:
Merge branch 'master' of https://github.com/microsoft/azure-docs-pr into luis-resources-faq

itechedit committed:
edit pass: luis-resources-faq

tdykstra committed:
change connection string wording

tdykstra committed:
minor edit

cephalin committed:
rewrite plan article

carolinacmoravia committed:
C6590: Broken metadata due to extra space

Hello, @Barclayn,

there is an extra space at the begging of the metadate that is causing broken format in localized version.

Could you help to merge this PR, please? Many thanks.

carolinacmoravia committed:
C6867: Hardcoded EN locale in link 

Hello, @ahmedelnably,
Reported links are hardcoded so they can't lead to target laguages for localized versions. Could you help to merge this PR, please?
Many thanks in advance.

cephalin committed:
staged review

cephalin committed:
acrolinx

cephalin committed:
staged review

tdykstra committed:
settings --> config

tdykstra committed:
fix link

tdykstra committed:
typo

mmacy committed:
[aci] propagate env vars to cmds

kirankumarkolli committed:
SDK 1.8.1 release note

@mimig1, @rnagpal please review

kirankumarkolli committed:
Updated comment

Commit:
add note about performance counters table

cephalin committed:
cory feedback

cephalin committed:
acrolinx

cephalin committed:
validation

cephalin committed:
acrolinx

mumian committed:
rename a cluster type

Commit:
custom-search: tutorial

Commit:
Added cross-sub registration.

Commit:
Fix

Commit:
custom-search: update toc

tdykstra committed:
Jeff H. feedback, other cleanup

tdykstra committed:
Jeff H. feedback

tdykstra committed:
fix links

Commit:
Formatting fixes.

rastala committed:
Update how-to-use-jupyter-notebooks.md

rastala committed:
Update how-to-use-jupyter-notebooks.md

itechedit committed:
Update luis-resources-faq.md

tdykstra committed:
fix links

rastala committed:
Update how-to-use-jupyter-notebooks.md

Commit:
updating stale data ingesting and exploration topics

rastala committed:
Update how-to-use-jupyter-notebooks.md

HeidiSteen committed:
Doc limit changes, portal provisioning change

Commit:
updating another stale data topic

Commit:
acrolinx edits

HeidiSteen committed:
scrubbed screenshot, text

douglaslMS committed:
Renamed Get Started topic as Setup, fixed links to it.

HeidiSteen committed:
doc limit edits

HeidiSteen committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into heidist-master

spelluru committed:
freshness review

spelluru committed:
changed author

spelluru committed:
acrolynx

identityMy committed:
Updating to remove preview text

PRMerger17 committed:
Merge pull request #27330 from jimdial/patch-30

Added cross-sub registration.

christiankuhtz committed:
updated reference for multi-vip

to newer multi-vip overview doc

PRMerger12 committed:
Merge pull request #27280 from mmacy/aci-mount-env-vars

[aci] propagate env vars to cmds

PRMerger17 committed:
Merge pull request #27381 from identityMy/master

Updating to remove preview text

PRMerger20 committed:
Merge pull request #27353 from bradsev/update-stale-tdsp-data-topics

updating stale data ingesting and exploration topics

rmca14 committed:
Merge pull request #25992 from tdykstra/cscstg

Add precompiled C# to storage binding docs

PRMerger10 committed:
Merge pull request #27310 from SergeyKanzhelev/sergkanz/addNoteAboutPerfCounters

add note about performance counters table

Commit:
Merge pull request #27376 from spelluru/adffreshness1109

freshness review

PRMerger17 committed:
Merge pull request #27383 from christiankuhtz/patch-33

updated reference for multi-vip

spelluru committed:
maximum limit on timeout is 90 days

tynevi committed:
Merge pull request #25615 from brapel/custom-search-tutorial

custom-search: tutorial

tynevi committed:
Merge pull request #27317 from mumian/djnewname1109

rename a cluster type

GitHubber17 committed:
Merge pull request #27344 from rastala/master

Updating AzureML preview Notebooks doc with cell magics

linda33wj committed:
Fix a property name

linda33wj committed:
Add support for logging faulty rows in ADLS

linda33wj committed:
Minor tweak

PRMerger16 committed:
Merge pull request #27420 from linda33wj/master

Refine ADF copy content

linda33wj committed:
Remove the empty copy activity security topic

JYOTHIRMAISURI committed:
mincor updates to ITSMC overview article

dominicbetts committed:
Update dates to release date for V2

dominicbetts committed:
Update dates to release date for V2

dominicbetts committed:
Update dates to release date for V2

PRMerger17 committed:
Merge pull request #27436 from dominicbetts/suite-reset-dates-2

Update dates to release date for V2

PRMerger14 committed:
Merge pull request #27435 from dominicbetts/suite-reset-dates-1

Update dates to release date for V2

Commit:
Merge pull request #27437 from dominicbetts/suite-reset-dates-3

Update dates to release date for V2

Commit:
freshness

PRMerger17 committed:
Merge pull request #27440 from Blackmist/freshness

freshness

CarlRabeler committed:
Update sql-database-security-tutorial.md

Commit:
Merge pull request #27441 from CarlRabeler/patch-361

Update sql-database-security-tutorial.md

HeidiSteen committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into heidist-master

cephalin committed:
stefan feedback

HeidiSteen committed:
rolled back the index limit change

PRMerger19 committed:
Merge pull request #27308 from kirankumarkolli/patch-5

SDK 1.8.1 release note

PRMerger12 committed:
Merge pull request #27433 from JYOTHIRMAISURI/master

minor updates to ITSMC overview article

PRMerger16 committed:
Merge pull request #27400 from spelluru/adfuntillimits1109

maximum limit on timeout is 90 days

rmca14 committed:
Merge pull request #27082 from cephalin/plan

rewrite plan article

PRMerger17 committed:
Merge pull request #27365 from douglaslMS/sync-rename-setup

Renamed Get Started topic as Setup, fixed links to it.

rjagiewich committed:
Merge pull request #27354 from HeidiSteen/heidist-master

Doc limit changes, portal provisioning change

v-alje committed:
Resolving merge conflict

PRMerger16 committed:
Merge pull request #27089 from carolinacmoravia/patch-424

C6590: Broken metadata due to extra space

ShawnJackson committed:
Merge pull request #26516 from itechedit/luis-resources-faq

edit pass: luis-resources-faq

PRMerger17 committed:
Merge pull request #27098 from carolinacmoravia/patch-426

C6887: Hardcoded EN locale in link

v-alje committed:
Merge pull request #27448 from MicrosoftDocs/FromPublicRepo

From public repo

tfosmark committed:
Merge pull request #27425 from linda33wj/master

Remove the empty copy activity security topic
